src/azure-cli/azure/cli/command_modules/acs/commands.py

```diff
@@ -233,3 +233,20 @@ def load_command_table(self, _):
             'rollback',
             'aks_mesh_upgrade_rollback',
             supports_no_wait=True)
+
+    # AKS approuting commands
+    with self.command_group(
+        'aks approuting', managed_clusters_sdk, client_factory=cf_managed_clusters
+    ) as g:
```

Comment:
```suggestion
    with self.command_group('aks approuting', managed_clusters_sdk, client_factory=cf_managed_clusters) as g:
```

Reply:
removed the line breaks in the command definitions